dev/sparktestsupport/shellutils.py

```diff
@@ -48,7 +48,10 @@ def run_cmd(cmd):
     if not isinstance(cmd, list):
         cmd = cmd.split()
     try:
-        subprocess.check_call(cmd)
+        if return_output:
+            return subprocess.check_output(cmd)
```

Comment:
I think that there was a workaround for this in fdcad6ef48a9e790776c316124bd6478ab6bd5c8; maybe we can take the workaround from that patch and move it into the `sparktestsupport` module so that we can use it from both scripts?